### DISCUSSION TITLE
Tailwind "breaking" some components?

### DISCUSSION THREAD

**Discussion by pablojsx** [tailwindlabs/tailwindcss#12395] [Help] [answered]
I just started to refactor a project which is a small e-commerce and I'm using Tailwind + NextUI + some CSS modules in a Vite app. When I try to use some NextUI components they look bad(broken), inputs look normal not sure why, but more important are my paypal buttons which look like they have a white background when it should be just 2 pill buttons, you can see them here: https://www.ez-tone.com/cart

When I deactivate the nextui() plugin in the tailwind config or comment the "@tailwind utilities" line in the index.css file the button look normal again.

Any clue??

**Comment by wongjn** [comment, marked as answer]
It seems to be the `color-scheme: dark` declaration in the `.dark, [data-theme=dark]` rule generated by Next UI in what seems to be the *Utilities* layer causing the white background. Applying `color-scheme: light` to the `<iframe>` element that contains the buttons removes the white background.

**Reply by pablojsx** [reply]
Thanks for the fast response! and yeah you are right, I added the "light" class to the paypal button and worked!
